### PR TITLE
[TESTMERGE TESTING]Hard Sync

### PR DESCRIPTION
**Pull request by kevinz000**
tin.

### CODE REVIEW

File: code/__defines/damage_organs.dm [hunk 1]
```diff
@@ -52,10 +52,10 @@
 #define ORGAN_NANOFORM 4 // VOREStation Add - Fully nanoswarm organ
 
 //Germs and infections.
-#define GERM_LEVEL_AMBIENT  30 // Maximum germ level you can reach by standing still.		//CITADEL CHANGE - reduces this value from 110 to 30 to make infections harder to get
-#define GERM_LEVEL_MOVE_CAP 110 // Maximum germ level you can reach by running around.	//CITADEL CHANGE - reduces this value from 200 to 110 to make infections harder to get
```
Comment (deathride58) [inline code/__defines/damage_organs.dm]: *cough

File: code/controllers/master.dm [hunk 2]
```diff
@@ -188,7 +202,6 @@ var/datum/controller/master/Master = new()
 	#endif
 	world.fps = config.fps
 	var/initialized_tod = REALTIMEOFDAY
-	world.TgsInitializationComplete()	//CIT CHANGE - adds hooks necessary for TGS3 integration
```
Comment (deathride58) [inline code/controllers/master.dm]: ???

Comment (kevinz000) [reply code/controllers/master.dm]: oops

**Comment by Zeldazackman** [comment]
P A N I C 
its-happening-scale

**Pull request merged**